aptos-move/e2e-move-tests/src/tests/aggregator_v2_events.rs

```diff
@@ -0,0 +1,237 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::{
+    assert_success,
+    tests::{aggregator_v2::EAGGREGATOR_OVERFLOW, common},
+    MoveHarness,
+};
+use aptos_language_e2e_tests::account::Account;
+use aptos_types::{
+    on_chain_config::FeatureFlag,
+    transaction::{ExecutionStatus, SignedTransaction, TransactionOutput},
+};
+use claims::{assert_matches, assert_ok, assert_some};
+use move_core_types::{
+    account_address::AccountAddress, ident_str, language_storage::ModuleId,
+    parser::parse_struct_tag, vm_status::AbortLocation,
+};
+use serde::{Deserialize, Serialize};
+
+#[derive(Deserialize, Serialize)]
+struct Aggregator {
+    value: u64,
+    max_value: u64,
+}
+
+#[derive(Deserialize, Serialize)]
+struct AggregatorSnapshot {
+    value: u64,
+}
+
+#[derive(Deserialize, Serialize)]
+struct Counter {
+    value: Aggregator,
+}
+
+#[derive(Deserialize, Serialize)]
+struct Event {
+    value: AggregatorSnapshot,
+}
+
+fn publish_test_package(h: &mut MoveHarness, aptos_framework_account: &Account) {
+    let path_buf = common::test_dir_path("aggregator_v2.data/pack");
+    assert_success!(h.publish_package_cache_building(aptos_framework_account, path_buf.as_path()));
+}
+
+fn create_test_txn(
+    h: &mut MoveHarness,
+    aptos_framework_account: &Account,
```

Comment:
Does this have to 0x1 account? Doesn't any account work?

Reply:
Any should work, no specific reason why I used 0x1